.github/workflows/test_build.yml

```diff
@@ -22,3 +22,13 @@ jobs:
       - name: Install wheel
         run: |
           pip3 install dist/torchmultimodal*.whl
+      - name: Test import
+        run: |
+          python -c "import torchmultimodal"
+      - name: Install dependencies
```

Comment:
We cant check import for all dependencies we add right? but the unit test checks that in some way
it will halt the flow anyways

Reply:
https://github.com/pytorch/torchrec/blob/main/.github/workflows/nightly_build_cpu.yml#L93-L135

Reply:
It would be too late to check in unit testing. Installing dependencies is also something one would do first when developing locally. [TorchRec](https://github.com/pytorch/torchrec/blob/main/.github/workflows/nightly_build_cpu.yml#L93-L135) follows the same sequence.

RE: how to verify imports: verify a few major upstreams would be good: torchtext, torchvision, pytorch